homework/src/index.js

```diff
@@ -1,47 +1,119 @@
-'use strict';
-
+"use strict";
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+  function main() {
+    fetchJSON("https://api.github.com/orgs/HackYourFuture/repos", (err, data) => {
+      function rend(changeEvent) {
+        main.innerHTML = "";
+        let sectL = crnap("section", main, { class: "left", id: "maininfo" });
+        const sectR = crnap("section", main, { class: "right", id: "contribInfo" });
+        let sectLinfobox = crnap("div", sectL, { id: "info-box", class: "scy" });
+        sectL = crnap("div", sectLinfobox, {
```

Comment:
sectR means section right

>(whatever it means)

You can understand it by "section class right" described inside of this call. (it's 1 extra sentence to read). I think there is no point to name it sectionRight or rightSection or similar to it, because it's just 1 line of code.